packages/core/package.json

```diff
@@ -0,0 +1,117 @@
+{
+  "name": "@inrupt/solid-client-authn-browser",
+  "version": "0.0.1-featsetup-lerna-238400657-650.0",
```

Comment:
Shouldn't the prerelease versions be only set in the CD pipeline ?

Reply:
Yeah, this must have been a temporary thing that I committed without realising - it's good in the code now!